### ISSUE TITLE
Request for missing VGGT preprocessing, RGB-X pipeline and scene metadata

### ISSUE THREAD

**Issue by Arkos123** [GrumpySloths/SGS_Intrinsic.github.io#3]
Hello, thank you very much for releasing the code! The lighting and material decomposition results shown in the paper are very impressive.

I am currently trying to reproduce the Mip-NeRF 360 `room` scene. I filled in some of the missing dependencies myself, and the training pipeline can now run successfully. 

However, the lighting/material decomposition results I obtain do not seem very good: the environment light is almost completely black, the point light is heavily overexposed, and the albedo also appears somewhat blurry.

<img height="500" alt="Image" src="https://github.com/user-attachments/assets/998b0a4e-71f2-4a8e-90d3-aed9be1904d3" />

My reproduction setup:
> - Scene: Mip-NeRF 360 `room
> - Image directory: `images_4`
> - Data split: 24 training views / 39 held-out views
> - Environment:
>     - Python 3.10.20
>     - PyTorch 2.1.2 + CUDA 11.8
>     - torchvision 0.16.2
> - Stage I: trained for 7,000 iterations
> - Stage II: resumed from Stage I iteration 7,000 and continued training until iteration 10,000
> - Training parameters are based on `scrips/train_mipnerf.sh`, with only the scene changed from `kitchen` to `room`.
> - Since the repository does not include the VGGT preprocessing code or several custom modules, I reimplemented the VGGT data conversion pipeline locally and replaced some unavailable dependencies with their corresponding public upstream implementations.

The albedo effect in the paper is significantly better.

<img width="444" height="330" alt="Image" src="https://github.com/user-attachments/assets/25b61764-4587-4022-a9db-a8121ae55715" />

I also tried making a few adjustments on my side, but the results are still not satisfactory.

In addition, the repository currently seems to be missing several pieces that may be important for reproducing the paper results. If possible, could you please provide the following materials when convenient? 

## 1. VGGT initialization

The training script uses `mipnerf360_vggt_v3`, but the repository does not include the VGGT preprocessing code used to generate this data.

Could you please provide more details, for example:

- The VGGT point-cloud initialization and data conversion scripts
- The VGGT model/checkpoint used (I am currently using the official VGGT release)
- The procedure used to select the 24 training views, or the exact image list
- Whether the final pipeline uses camera parameters predicted by VGGT or the original Mip-NeRF/COLMAP cameras
- Whether the VGGT point cloud is aligned to the Mip-NeRF camera coordinate system, and whether any scale normalization is applied

## 2. Missing RGB-X pipeline

[utils/rgbx_guidance.py](https://github.com/GrumpySloths/SGS_Intrinsic.github.io/blob/282f76608dc24ea217ae9eebb05d2141ceae00c2/utils/rgbx_guidance.py#L1)
imports:

```python
from rgb2x.pipeline_rgb2x_myversion import StableDiffusionAOVMatEstPipeline
```

However, `pipeline_rgb2x_myversion.py` is not included in the repository.

Could you please upload this file, or provide the RGB-to-X fork used in the experiments?

## 3. Custom dependencies

`requirements.txt` contains several local paths such as `file:///home/jiahao/...`.

Although these dependencies can be replaced with their public repositories, I am not sure whether the paper experiments used locally modified versions or custom forks.

## 4. Semantic categories

The paper uses Qwen3-VL to obtain scene categories.

Could you please provide the category list used for the Mip-NeRF 360 `room` scene?

Thank you very much!  🙏
